### DISCUSSION TITLE
Documentation enhancement

### DISCUSSION THREAD

**Discussion by AktasC** [Jyers/Marlin#898] [Ideas]
Hello world,

I've been using Jyers' firmware for two weeks now, started with the default and slowly upgraded till I got to UBL + BL Touch .
(So, first of all, thanks for the build I really appreciate it)  

I feel like the sole wiki isn't enough to cover and explain how this works, I'm mainly speaking about UBL here.
I've had to go and look at Marlin's official documentation to understand how it worked and what Start & End G-Code I had to have.

Also, using the 5x5 grid while the X & Y probe offsets aren't accurate enough doesn't seem to be as big of an issue as using the 10x10 grid _which is logical_
I feel like that should be a **BIG BOLD WARNING** in the description on the wiki to avoid hours and hours of frustration :)

Furthermore, I feel like we could also, as a community, try and create a better documentation for this firmware 🚀 
With over 14k forks, I feel like it deserves something clean like a gitbook or notion md at least
I'm pretty certain not to be the only dev in this community and can't wait to hear more about your ideas and PoV :) 

What do you think ?

**Comment by Jyers** [comment]
I totally agree more documentation would be ideal. Even just making the wiki a bit more robust would be a significant benefit. I have had several people reach out about adding new info or improving what's there, but so far no one has gotten as far as to send me a draft. I would happily support any intent to create better documentation for the firmware.

PS - The probe offset thing... That was why I used to have it defaulted to -10, -10. So people would have to measure it themselves. But then it became an issue when no one understood they had to set it themselves, so it was put pack to a decent approximation for most mounts.

**Reply by AktasC** [reply]
I'll see what I can do :) 

Is there any way you could DM me or add me on Discord or something ?
Might come in handy @Jyers

**Reply by Jyers** [reply]
You can reach me at my email Jyers.Marlin@gmail.com or on discord at Jyers#4194

**Comment by Tuncay-Ayhan** [comment]
> Hello world,
> 
> 
> 
> I've been using Jyers' firmware for two weeks now, started with the default and slowly upgraded till I got to UBL + BL Touch .
> 
> (So, first of all, thanks for the build I really appreciate it)  
> 
> 
> 
> I feel like the sole wiki isn't enough to cover and explain how this works, I'm mainly speaking about UBL here.
> 
> I've had to go and look at Marlin's official documentation to understand how it worked and what Start & End G-Code I had to have.
> 
> 
> 
> Also, using the 5x5 grid while the X & Y probe offsets aren't accurate enough doesn't seem to be as big of an issue as using the 10x10 grid _which is logical_
> 
> I feel like that should be a **BIG BOLD WARNING** in the description on the wiki to avoid hours and hours of frustration :)
> 
> 
> 
> Furthermore, I feel like we could also, as a community, try and create a better documentation for this firmware 🚀 
> 
> With over 14k forks, I feel like it deserves something clean like a gitbook or notion md at least
> 
> I'm pretty certain not to be the only dev in this community and can't wait to hear more about your ideas and PoV :) 
> 
> 
> 
> What do you think ? 

Just a little note.

14K forks are from the Marlin upstream repo. 

Jyers/Marlin has 121 forks (at this moment) and 593 stars.

![image](https://user-images.githubusercontent.com/28215053/119382740-c03f8e00-bcc2-11eb-8efa-744984f42362.png)

**Reply by AktasC** [reply]
I don't see how that matters or changes anything but thank you I guess 👍

**Reply by Tuncay-Ayhan** [reply]
It doesn't. But the difference between it was huge. Just wanted to point it out, not trying to be a smartass or anything 🤪
